src/content/docs/en/reference/cli-reference.mdx

```diff
@@ -287,6 +288,48 @@ Adapter                  @astrojs/vercel/serverless
 Integrations             none
 ```
 
+## `astro preferences`
+
+Manage user preferences with the `astro preferences` command. User preferences are specific to individual Astro users, unlike the `astro.config.mjs` file which changes behavior for everyone working on a project.
+
+User preferences are scoped to the current project by default, stored in a local `.astro/settings.json` file. Using the `--global` flag, user preferences can also be applied to every Astro project on the current machine. Global user preferences are stored in an operating system-specific location.
+
+For example, to disable the devToolbar in a specific Astro project:
+
+```shell
+astro preferences disable devToolbar
+```
+
+To disable the devToolbar in all Astro projects on the current machine:
+
+```shell
+astro preferences disable --global devToolbar
+```
+
+The devToolbar can later be enabled with:
+
+```shell
+astro preferences enable devToolbar
+```
+
+The `reset` command resets a preference to its default value:
+
+```shell
+astro preferences reset devOverlay
+```
+
+The `list` command prints all current user preferences. It also supports a machine-readable `--json` output.
```

Comment:
```suggestion
The `list` command prints the current settings of all configurable user preferences. It also supports a machine-readable `--json` output.
```
If we just add this, then I don't think we need a section about preferences at all! So, just make this true! lol